lib/classes/class-settings.php

```diff
@@ -127,6 +127,48 @@ public function __construct($args = false) {
           }
           $this->set('property_types', $ar);
         }
+        
+        /* Standard Attributes 
+         * added 28/07/2016 @raj
+         * 
+         */
+
+        $PDF_unique = array(
```

Comment:
@pemaxim I moved the json data to composer.json
get_schema seems to be working fine.

question:
1) It seems you pasted same link twice in your comment. Did you mean to provide some other link ?
2) should I add localisations like  following link for the data coming from config.json ?
https://github.com/wp-property/wp-property/blob/feature-standard-attribute/lib/classes/class-bootstrap.php#L161-L167